source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/OverflowActionLayoutRenderer.java

```diff
@@ -0,0 +1,317 @@
+                                               ViewGroup actionButtonsLayout,
+                                               ViewGroup contentLayout,
+                                               ICardActionHandler cardActionHandler,
+                                               HostConfig hostConfig,
+                                               RenderArgs renderArgs) throws AdaptiveFallbackException
+    {
+        FeatureRegistration featureRegistration = CardRendererRegistration.getInstance().getFeatureRegistration();
+
+        for (BaseActionElement actionElement : elementVector)
+        {
+            IBaseActionElementRenderer actionRenderer = CardRendererRegistration.getInstance().getActionRenderer(AdaptiveCardObjectModel.ModeToString(Mode.Secondary));
+
+            try
+            {
+                if (actionRenderer == null)
+                {
+                    throw new AdaptiveFallbackException(actionElement);
+                }
+
+                if ((featureRegistration != null) && (!actionElement.MeetsRequirements(featureRegistration)))
+                {
+                    throw new AdaptiveFallbackException(actionElement, featureRegistration);
+                }
+
+                contentLayout.addView(actionRenderer.render(renderedCard, context, fragmentManager, actionButtonsLayout, actionElement, cardActionHandler, hostConfig, renderArgs));
+            }
+            catch (AdaptiveFallbackException e)
+            {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    /**
+     * Gets color for a given color attribute.
+     *
+     * @param context        context.
+     * @param colorAttribute attribute name.
+     * @return Id of a color resource or -1 id resource not found.
+     */
+    @ColorInt
+    public static int getAttributeColor(Context context, @AttrRes int colorAttribute)
+    {
+        int[] attrs = {colorAttribute};
+        TypedArray typedArray = context.obtainStyledAttributes(attrs);
+        @ColorInt int color = ContextCompat.getColor(context, typedArray.getResourceId(0, -1));
+        typedArray.recycle();
+        return color;
+    }
+
+    private static OverflowActionLayoutRenderer s_instance = null;
+
+    /**
+     * Responsible for default Overflow rendering behavior.
+     */
+    public static class OverflowActionRenderer implements IOverflowActionRenderer {
+
+        private static OverflowActionRenderer m_innerInstance = null;
+
+        public static OverflowActionRenderer getInstance()
+        {
+            if (m_innerInstance == null)
+            {
+                m_innerInstance = new OverflowActionRenderer();
+            }
+
+            return m_innerInstance;
+        }
+
+
+        @Override
+        public boolean onRenderOverflowAction(@NonNull BaseActionElementVector actionElements, @NonNull ViewGroup viewGroup, boolean isRootLevelActions)
+        {
+            return false;
```

Comment:
Any reason we can't call the actual code that renders the overflow button from this method?

This would allow a host to simply `extend` this class to make slight modifications to the behavior, and then they just register their subclass.

That's the architecture we seem to follow elsewhere (e.g. `ActionLayoutRenderer` extends `IActionLayoutRenderer`, and it does the actual rendering).

Reply:
This implementation is not required in the SDK, because default implementation always will be rending the `"..."` button. Hence implementing `IOverflowActionRenderer` in the SDK is unnecessary and it is removed. 
However, if client wants to customize rendering behavior, this callback _(return type changed to View)_ will be invoked by `SDK` during `Overflow` action view rendering and client can either create a customized `view` or let the SDK render the default `"..."` view. So, client can have combination of both default & their own view.

On an another note, If client wants to render entire `OverflowLayout` themselves then can register their `OverflowLayoutRenderer` _(method added in CardRegistration)_ like they are doing it for `ActionLayoutRenderer`